src/views/v2/components/ical-link.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Component: iCal Link
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/components/ical-link.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var object $ical Object containing iCal data
+ */
+
+if ( empty( $ical->display_link ) ) {
+	return;
+}
+
+?>
+<div class="tribe-events-c-ical tribe-common-b1">
+	<a
+		class="tribe-common-anchor-alt"
```

Comment:
can you add back the `tribe-events-c-ical__link` class, to allow users to hook into it if they want to make changes? so we have both `tribe-common-anchor-alt` and `tribe-events-c-ical__link` classes.